docs/contributing/dependencies/index.md

```diff
@@ -50,15 +50,15 @@ change. For more details read
 
 ## Workflow
 
-Renovate will automatically create pull requests during the weekend, which naturally aligns with
-each team allocating some time during the following Monday to work through their respective queue of
-pull requests. The teams should work together to resolve outstanding pull requests within the week
-to avoid stagnation.
+Renovate is currently scheduled to automatically create pull requests every 2 weeks for all open
+dependencies. The goal of our dependency management process is for the teams to handle these
+dependencies in that same 2-week cadence, in order to avoid a large backlog of PRs and out-of-date
+packages building up over time.
```

Comment:
What is an `open dependency`? 

```suggestion
Renovate is currently scheduled to automatically create pull requests every 2 weeks. The goal of our
dependency management process is for the teams to review and merge the opened pull requests in the
same 2-week cadence. To avoid a large backlog of PRs and out-of-date packages accumulating.
```

Reply:
Updated to use your suggestion in [`4176743` (#304)](https://github.com/bitwarden/contributing-docs/pull/304/commits/4176743c791a5540e87072b82009019827368f03).